code/src/App.js

```diff
@@ -1,12 +1,17 @@
 import React from 'react'
-import data from './data.json'
+import data from 'data.json'
 
-console.log(data)
+import MusicInfo from 'components/MusicInfo'
+import Header from 'components/Header'
+
+/* Rendering the main structure of website */
 
 export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
+	return (
+    <div className="wrapper">
+      <header><Header /></header>
```

Comment:
Nice use of a Header component! Maybe you could also put the header html tags in the component?